x-pack/test_serverless/api_integration/test_suites/security/config.ts

```diff
@@ -25,5 +26,11 @@ export default createTestConfig({
     '--coreApp.allowDynamicConfigOverrides=true',
     `--xpack.securitySolutionServerless.cloudSecurityUsageReportingTaskInterval=5s`,
     `--xpack.securitySolutionServerless.usageApi.url=http://localhost:8081`,
+    '--xpack.dataUsage.enabled=true',
```

Comment:
```suggestion
    '--xpack.dataUsage.autoops.enabled=true',
```

Reply:
I cannot make it without `autoops` prefix

Reply:
I added `--xpack.dataUsage.autoops.enabled=true`, though left `--xpack.dataUsage.enabled=true` in place so as not to enable the plugin.